content/08.outlook.md

```diff
@@ -1 +1,45 @@
+
+In addition to sample scarcity, there is a dearth of comprehensive phenotypic-genotypic databases in rare disease.
+With the ubiquity of sequencing platforms, genomic data has been easy to gather for rare disease patients.[@doi:10.1038/nrg3555; @doi:10.1038/nrg.2017.116; @doi:10.1056/NEJMra1711801]
+An important next step would be to integrate genomic and clinical data to produce comprehensive phenotypic-genotypic databases that can fuel interpretation of features extracted using ML methods.
+Finally, mindful sharing of data with proper metadata and attribution to enable prompt data reuse is of utmost important in building datasets that can be of great value in rare disease. [@https://www.nature.com/articles/s41576-020-0257-5]
+
+
+_Development of methods that reliably support mechanistic interrogation of specific rare diseases_
+
+A pattern that emerged readily in our quest for examples of application of ML methods in rare disease is that most ML related strategies were focused on classification of a patient as a rare disease patient among other non-rare disease patients.
+We found that examples of methodologies interrogating mechanistic details of a specific rare disease were particularly lacking.
+Reasons for this include not only the lack of large datasets, but also lack of reliability or interpretability of the results from such analyses.
+We believe that an intentional push towards developing methods or analytical workflows that address this would be critical in encouraging powerful analyses using rare disease data.
+
+Method development with rare disease applications in mind requires the developers to bear the responsibility of ensuring that the resulting model is _trustworthy_.
+The field of natural language processing has a few examples of how this can be achieved.[@https://www.aclweb.org/anthology/N16-3020.pdf, @https://www.aclweb.org/anthology/P19-1073.pdf]
+One way to increase user's trust on a developed model is by helping the user understand the behavior of the developed model through providing explanations regarding why a certain model made certain predictions.[@https://www.aclweb.org/anthology/N16-3020.pdf]
+Another approach is to provide robust _error analysis_ for newly developed models to help users understand the strengths and weaknesses of a model.[@https://www.aclweb.org/anthology/P19-1073.pdf; @https://www.mitpressjournals.org/doi/abs/10.1162/COLI_a_00072; @doi:10.1093/bioinformatics/bth060]
+Adoption of these kind of approaches into biological data and analysis is still rare but is quickly becoming necessary as machine learning approaches become mainstream in biomedicine.
+
+Finally, methods that can reliably bring disparate datasets together will always remain a need of rare diseases.
```

Comment:
```suggestion
Finally, methods that can reliably integrate disparate datasets will always remain a need of rare diseases.
```

Reply:
why?

Reply:
My thoughts are as follows:
1. Since rare disease patients will be sparse by the virtue of rarity of disease...Even if we can ensure that every NF patient sample is collected in a standardized manner, patients in UK will be processed differently... 
2. There will always be temporal differences in sample collection and processing, due to the nature of clinical practice. In diseases that are common, samples with significant temporal differences can be disregarded from an analysis, but that luxury is not available to the rare disease field.